Python/bytecodes.c

```diff
@@ -2147,18 +2147,23 @@ dummy_func(
             PyObject *owner_o = PyStackRef_AsPyObjectBorrow(owner);
             assert(Py_TYPE(owner_o)->tp_dictoffset < 0);
             assert(Py_TYPE(owner_o)->tp_flags & Py_TPFLAGS_INLINE_VALUES);
-            DEOPT_IF(!_PyObject_InlineValues(owner_o)->valid);
+            DEOPT_IF(!FT_ATOMIC_LOAD_UINT8(_PyObject_InlineValues(owner_o)->valid));
         }
 
         split op(_LOAD_ATTR_INSTANCE_VALUE, (offset/1, owner -- attr, null if (oparg & 1))) {
             PyObject *owner_o = PyStackRef_AsPyObjectBorrow(owner);
             PyObject **value_ptr = (PyObject**)(((char *)owner_o) + offset);
-            PyObject *attr_o = *value_ptr;
+            PyObject *attr_o = FT_ATOMIC_LOAD_PTR_ACQUIRE(*value_ptr);
             DEOPT_IF(attr_o == NULL);
+            #ifdef Py_GIL_DISABLED
+            if (!_Py_TryIncrefCompareStackRef(value_ptr, attr_o, &attr)) {
```

Comment:
What does `_Py_TryIncrefCompareStackRef` do?
Function names need to be self explanatory. The result of this function does not depend on a comparison, but on whether something has been modified. Maybe `_Py_TryIncrefIfPointersConsistentStackRef` as it only works if the `PyObject ** and PyObject *pointers are consistent.

I know that `_Py_TryIncrefCompareStackRef` was not introduced in this PR, but it is critical to the understanding of this code.

Reply:
I still think "Compare" is misleading. The name should say what the function does, not how it does it.